### ISSUE TITLE
::Fixnum is deprecated warning on MRI

### ISSUE THREAD

**Issue by azumakuniyuki** [sisimai/rb-sisimai#268]
But JRuby requires `Fixnum` like the following:
```
  1) Failure:
MailMaildirTest#test_offset [rb-sisimai/test/public/mail-maildir-test.rb:77]:
Expected 527 to be an instance of Integer, not Fixnum.
```

See also https://qiita.com/jkr_2255/items/647c427d2c2f7892fa93

**Comment by azumakuniyuki** [comment]
JRuby 9.4 says `warning: constant ::Fixnum is deprecated`

**Comment by azumakuniyuki** [comment]
Sisimai 5 requires JRuby 9.2 or later, I've decided just now.

**Comment by azumakuniyuki** [comment]
This issue resolved at v5.0.2
